boskos/janitor/gcp_janitor.py

```diff
@@ -316,22 +338,28 @@ def main(project, days, hours, filt, rate_limit):
     err = 0
     age = datetime.datetime.utcnow() - datetime.timedelta(days=days, hours=hours)
     clear_all = (days is 0 and hours is 0)
-    for res in DEMOLISH_ORDER:
-        log('Try to search for %r with condition %r' % (res.name, res.condition))
-        try:
-            col = collect(project, age, res, filt, clear_all)
-            if col:
-                err |= clear_resources(project, col, res, rate_limit)
-        except (subprocess.CalledProcessError, ValueError):
-            err |= 1 # keep clean the other resource
-            print >>sys.stderr, 'Fail to list resource %r from project %r' % (res.name, project)
 
-    # try to clean leaking gke cluster
-    try:
-        err |= clean_gke_cluster(project, age, filt)
-    except ValueError:
-        err |= 1 # keep clean the other resource
-        print >>sys.stderr, 'Fail to clean up cluster from project %r' % project
+    if service_account != "" or os.environ["GOOGLE_APPLICATION_CREDENTIALS"] != "":
```

Comment:
oops.. this is python 😂 `if not service_account`, and I'd suggest make it default to `GOOGLE_APPLICATION_CREDENTIALS` to get rid of the or here